pkg/ansible/runner/runner.go

```diff
@@ -66,13 +66,15 @@ func ansibleVerbosityString(verbosity int) string {
 
 type cmdFuncType func(ident, inputDirPath string, maxArtifacts, verbosity int) *exec.Cmd
 
+//nolint:gosec //to be done later
 func playbookCmdFunc(path string) cmdFuncType {
 	return func(ident, inputDirPath string, maxArtifacts, verbosity int) *exec.Cmd {
 		return exec.Command("ansible-runner", ansibleVerbosityString(verbosity), "--rotate-artifacts",
 			fmt.Sprintf("%v", maxArtifacts), "-p", path, "-i", ident, "run", inputDirPath)
 	}
 }
 
+//nolint:gosec //to be done later
```

Comment:
This is not enabled in the CI.
To test the same lint checks locally which are made in the CI you can run `./hack/tests/check-lint.sh ci`

Reply:
Solved